Microsoft.Alm.Authentication/VstsAadAuthentication.cs

```diff
@@ -48,12 +48,15 @@ public sealed class VstsAadAuthentication : BaseVstsAuthentication, IVstsAadAuth
         /// access tokens acquired.</param>
         /// <param name="adaRefreshTokenStore">The secure secret store for storing any Azure tokens
         /// acquired.</param>
+        /// <param name="loginhint">An optional login username and/or domain for guiding non-interactive logon.</param>
         public VstsAadAuthentication(
             Guid tenantId,
             VstsTokenScope tokenScope,
-            ICredentialStore personalAccessTokenStore)
+            ICredentialStore personalAccessTokenStore,
+            string loginhint = null)
```

Comment:
Please no default parameters. If this is truly optional, just accept `null` as a valid argument (which seems to make the most sense. 

`AzureAuthority.NoninteractiveAcquireToken` gets away with it because it isn't publicly exported.